app/services/research_safety_adapters/scishield_training_synchronizer.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module ResearchSafetyAdapters
+
+  class ScishieldTrainingSynchronizer
+    def initialize
+      @users = User.active.select(:id, :email)
+    end
+
+    def synchronize
+      return if api_unavailable?
```

Comment:
I think we will want to know if the API is down - can you add a `Rollbar.error` here?